specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2636 @@
+      "properties": {
+        "sasUris": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "SAS URIs for the target blobs to import from",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUris"
+      ],
+      "description": "Parameters for a Redis Enterprise import operation."
+    },
+    "ExportClusterParameters": {
+      "title": "Export an RDB file into a target database",
+      "type": "object",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "SAS URI for the target directory to export to",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise export operation."
+    },
+    "OperationStatus": {
+      "description": "The status of a long-running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation's unique id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation's name."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the operation."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the operation."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse",
+          "description": "Error response describing why the operation failed."
+        }
+      }
+    },
+    "ForceLinkParameters": {
+      "type": "object",
+      "description": "Parameters for reconfiguring active geo-replication, of an existing database that was previously unlinked from a replication group.",
+      "required": [
+        "groupNickname",
+        "linkedDatabases"
+      ],
+      "properties": {
+        "groupNickname": {
+          "type": "string",
+          "description": "The name of the group of linked database resources. This should match the existing replication group name."
+        },
+        "linkedDatabases": {
+          "type": "array",
+          "description": "The resource IDs of the databases that are expected to be linked and included in the replication group. This parameter is used to validate that the linking is to the expected (unlinked) part of the replication group, if it is splintered.",
+          "items": {
+            "$ref": "#/definitions/LinkedDatabase"
+          }
+        }
+      }
+    },
+    "ForceUnlinkParameters": {
+      "title": "Forcibly unlink another database from this database.",
+      "description": "Parameters for a redis enterprise active geo-replication force unlink operation.",
```

Comment:
>redis enterprise [](http://example.com/codeflow?start=39&length=16)

We capitalized it Redis Enterprise elsewhere -- not that I want anyone to block on this, just calling it out.